assets/styles.css

```diff
@@ -0,0 +1,129 @@
+html{
+  width: 100vw;
+  height: 100vh;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #E3EDF7;
+  font-family: 'Exo 2', sans-serif;
+}
+
+.keyboard {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 50%;
+}
+
+.keyboard__text-area {
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.keyboard__message {
+  margin-top: 20px;
+}
+
+.keyboard-panel {

```

Comment:
empty style

Reply:
Removed empty styles in commit below

https://github.com/BlackTruth/virtual-keyboard/commit/88ba0949a558aa2d031f3d98f72f4d8880f72717